src/ZendDiagnostics/Check/Redis.php

```diff
@@ -55,16 +62,26 @@ private function createClient()
     {
         if (class_exists('\Redis')) {
             $client = new RedisExtensionClient();
-            $client->connect($this->host);
+            $client->connect($this->host, $this->port);
+
+            if ($this->auth) {
+                $client->auth($this->auth);
```

Comment:
Added exception for authorization check, but it's overhead, because Redis when password is incorrect throws RedisException and then test will return Failure